config/routes.rb

```diff
@@ -103,4 +105,8 @@
       as: "bibliographies_wole_soyinka", defaults: { section: "bibliographies" }
   end
 
+  #redefine default routes so they don't show up outside of the sections, where metadata can get confused
+  get 'item/:id', to: 'general#item_redir', as: 'item', constraints: { id: with_period }
+  get "search", to: 'general#index', as: "search"
```

Comment:
After further reflection I think for better SEO this would be better handled by the `redirects.yml` middleware doing a formal 301 redirect of URL requests with these patterns to the sites `/` URL. As is, these pages would return 200 HTTP responses and search engines would index any URLs matching these routes when instead we want the search engines to not index them as they should all be 301 redirected to the `/` home page.

Please drop these routes (and controller override) and put in full 301 redirects in `config/redirects.yml`. :pray:

Reply:
done